deploy/packages/emqx.service

```diff
@@ -5,13 +5,34 @@ After=network.target
 [Service]
 User=emqx
 Group=emqx
-Type=forking
+
+# The ExecStart= is foreground, so 'simple' here
+Type=simple
 Environment=HOME=/var/lib/emqx
-ExecStart=/usr/bin/emqx start
+
+# Enable logging to file
+Environment=EMQX_LOG__TO=default
+
+# Start 'foregroun' but not 'start' (daemon) mode.
+# Because systemd monitor/restarts 'simple' services
+ExecStart=/usr/bin/emqx foreground
+
+# Give EMQX enough file descriptors
 LimitNOFILE=1048576
```

Comment:
This is for tuning I think maybe we should remove it from here and let user override this limit via systeamd config .

Reply:
question,
do you mean `DefaultLimitNOFILE=` ? 
is it the system-wide default is it a good practice to give a big default for all services ?

Reply:
something like this 
```
[ec2-user@ip-172-31-35-122 emqx]$ cat /usr/lib/systemd/system/emqx.service.d/ulimit.conf
[Service]
LimitNOFILE=2000
```

Reply:
`emqx.service.d`

Reply:
we'll keep it as is for now